v63004/outref/gtm8202.txt

```diff
@@ -0,0 +1,113 @@
+##FILTERED##%YDB-I-MUJNLSTAT, End processing at ... ... .. ..:..:.. 20..
+#  search extract file for set variables
+2 ^a(2)="b"
+3 ^b(1)="1"
+4 ^b(2)="2"
+5 ^a(3)="c"
+5 ^a(4)="d"
+#  multi region extract -SEQNO=~1,2,~3
+#  expected to return sequences 2
+##FILTERED##%YDB-I-MUJNLSTAT, Initial processing started at ... ... .. ..:..:.. 20..
+%YDB-W-NOTALLJNLEN, Journaling disabled/off for one or more regions
+##FILTERED##%YDB-I-MUJNLSTAT, Backward processing started at ... ... .. ..:..:.. 20..
+##FILTERED##%YDB-I-MUJNLSTAT, Forward processing started at ... ... .. ..:..:.. 20..
+%YDB-I-FILECREATE, Journal extract file ./Reg.mjf created
+%YDB-S-JNLSUCCESS, Extract successful
+%YDB-S-JNLSUCCESS, Verify successful
+##FILTERED##%YDB-I-MUJNLSTAT, End processing at ... ... .. ..:..:.. 20..
+#  search extract file for set variables
+2 ^a(2)="b"
+#  multi region extract -SEQNO=~(~1,2,~3)
+#  expected to return sequences 1,3
+##FILTERED##%YDB-I-MUJNLSTAT, Initial processing started at ... ... .. ..:..:.. 20..
+%YDB-W-NOTALLJNLEN, Journaling disabled/off for one or more regions
+##FILTERED##%YDB-I-MUJNLSTAT, Backward processing started at ... ... .. ..:..:.. 20..
+##FILTERED##%YDB-I-MUJNLSTAT, Forward processing started at ... ... .. ..:..:.. 20..
+%YDB-I-FILECREATE, Journal extract file ./Reg.mjf created
+%YDB-S-JNLSUCCESS, Extract successful
+%YDB-S-JNLSUCCESS, Verify successful
+##FILTERED##%YDB-I-MUJNLSTAT, End processing at ... ... .. ..:..:.. 20..
+#  search extract file for set variables
+1 ^a(1)="a"
+3 ^b(1)="1"
+#  multi region extract -SEQNO=1,~2,3
+#  expected to return sequences 1,3
+##FILTERED##%YDB-I-MUJNLSTAT, Initial processing started at ... ... .. ..:..:.. 20..
+%YDB-W-NOTALLJNLEN, Journaling disabled/off for one or more regions
+##FILTERED##%YDB-I-MUJNLSTAT, Backward processing started at ... ... .. ..:..:.. 20..
+##FILTERED##%YDB-I-MUJNLSTAT, Forward processing started at ... ... .. ..:..:.. 20..
+%YDB-I-FILECREATE, Journal extract file ./Reg.mjf created
+%YDB-S-JNLSUCCESS, Extract successful
+%YDB-S-JNLSUCCESS, Verify successful
+##FILTERED##%YDB-I-MUJNLSTAT, End processing at ... ... .. ..:..:.. 20..
+#  search extract file for set variables
+1 ^a(1)="a"
+3 ^b(1)="1"
+#  multi region extract -SEQNO=~(1,~2,3)
+#  expected to return sequences 2
+##FILTERED##%YDB-I-MUJNLSTAT, Initial processing started at ... ... .. ..:..:.. 20..
+%YDB-W-NOTALLJNLEN, Journaling disabled/off for one or more regions
+##FILTERED##%YDB-I-MUJNLSTAT, Backward processing started at ... ... .. ..:..:.. 20..
+##FILTERED##%YDB-I-MUJNLSTAT, Forward processing started at ... ... .. ..:..:.. 20..
+%YDB-I-FILECREATE, Journal extract file ./Reg.mjf created
+%YDB-S-JNLSUCCESS, Extract successful
+%YDB-S-JNLSUCCESS, Verify successful
+##FILTERED##%YDB-I-MUJNLSTAT, End processing at ... ... .. ..:..:.. 20..
+#  search extract file for set variables
+2 ^a(2)="b"
+#  single region extract -SEQNO=7,8 TP transaction 
+#  expecting a.lost to be created
```

Comment:
We expect a.lost to be created. But the reference file below does not indicate a.lost being created.